pulp_container/app/authorization.py

```diff
@@ -22,18 +29,43 @@ class AuthorizationService:
     according to a user's scope.
     """
 
-    def generate_token(self, username, service, scope):
-        """
-        Generate a Bearer token.
+    ANONYMOUS_USER = "AnonymousUser"
+    VALID_ACTIONS = ["pull", "push", "*"]
 
-        A signed JSON web token is generated in this method. The structure of the token is
-        adjusted according the documentation https://docs.docker.com/registry/spec/auth/jwt/.
+    def __init__(self, username, service, scope):
```

Comment:
s/username/user

Reply:
how about we pass the user directly into the generate_token https://github.com/pulp/pulp_container/blob/rbac/pulp_container/app/registry_api.py#L327